README.md

```diff
@@ -9,3 +9,19 @@
 ---
 
 [![Build Status](https://travis-ci.org/justeat/global-component-library.svg)](https://travis-ci.org/justeat/global-component-library)
+
+## Usage
+
+To get up and running with the Fozzie Component Library, follow these steps:
+
+1. Clone this repo to your local machine `git clone https://github.com/justeat/global-component-library.git`
+2. In your command prompt, run `yarn` from the base of the project. This will install all of the necessary project dependencies.
```

Comment:
Should we include a step about installing yarn. Same for the next one, step for gulp?

Reply:
I'm working up another docs PR now – so will add this in